Prox/Prox/RemoteConfigDefaults.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>place_categories_to_hide_csv</key>
+	<string></string>
```

Comment:
For now, yes – we don't know for sure what categories we want to filter so we should start at nothing.